toolkit/tools/imagecustomizerapi/packagelist.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
```

Comment:
The existing tools support two inputs, `PackageLists` (path to .txt files) and `Packages` (list of package names), thoughts on continuing to support both here?